cumulus/test/runtime/src/lib.rs

```diff
@@ -350,6 +352,45 @@ impl pallet_glutton::Config for Runtime {
 	type WeightInfo = pallet_glutton::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	pub storage BlocksPerPoV: u32 = 1;
+}
+
+/// Super ultra hacky core selector.
+///
+/// TODO: Replace with something useful.
```

Comment:
Would you explain a bit more?

Reply:
This code right now is ultra shitty. Just to make tests work. This entire core selection logic is highly flawed and requires some reworking when you want to put multiple blocks into one `PoV`. Before it was yoloing the last byte of the block number into the selector, which is not working if you use multiple blocks per pov (because every block would return a different selector).